backend/tests/v5/e2e/routes/teamspaces/projects/containers/containers.test.js

```diff
@@ -173,6 +182,125 @@ const testGetContainerStats = () => {
+		});
+
+		test('should fail if the user is not a member of the teamspace', async () => {
+			const res = await agent.patch(`${route(voidRevision._id)}?key=${nobody.apiKey}`)
+				.send({ void: false }).expect(templates.teamspaceNotFound.status);
+			expect(res.body.code).toEqual(templates.teamspaceNotFound.code);
+		});
+
+		test('should fail if the project does not exist', async () => {
+			const res = await agent.patch(`/v5/teamspaces/${teamspace}/projects/dflkdsjfs/containers/${modelWithRev._id}/revisions/${voidRevision._id}?key=${users.tsAdmin.apiKey}`)
+				.send({ void: false }).expect(templates.projectNotFound.status);
+			expect(res.body.code).toEqual(templates.projectNotFound.code);
+		});
+
+		test('should fail if the user does not have access to the container', async () => {
+			const res = await agent.patch(`${route(voidRevision._id)}?key=${users.noProjectAccess.apiKey}`)
+				.send({ void: false }).expect(templates.notAuthorized.status);
+			expect(res.body.code).toEqual(templates.notAuthorized.code);
+		});
+
+		test('should fail if the container doesn\'t exist', async () => {
+			const res = await agent.patch(`/v5/teamspaces/${teamspace}/projects/${project.id}/containers/dfsfaewfc/revisions/${voidRevision._id}?key=${users.tsAdmin.apiKey}`)
+				.send({ void: false }).expect(templates.containerNotFound.status);
+			expect(res.body.code).toEqual(templates.containerNotFound.code);
+		});
+
+		test('should fail if the revision doesn\'t exist', async () => {
+			const res = await agent.patch(`/v5/teamspaces/${teamspace}/projects/${project.id}/containers/${modelWithRev._id}/revisions/fdefaxsxsa?key=${users.tsAdmin.apiKey}`)
+				.send({ void: false }).expect(templates.revisionNotFound.status);
+			expect(res.body.code).toEqual(templates.revisionNotFound.code);
+		});
+
+		test('should not update a revision\'s status if the body if the request is not boolean', async () => {
+			await agent.patch(`${route(voidRevision._id)}?key=${users.tsAdmin.apiKey}`)
+				.send({ void: 123 }).expect(templates.ok.status);
+
+			const res = await agent.get(`/v5/teamspaces/${teamspace}/projects/${project.id}/containers/${modelWithRev._id}/revisions?showVoid=true&key=${users.tsAdmin.apiKey}`)
+				.expect(templates.ok.status);
+			const revision = res.body.find((r) => r._id === voidRevision._id);
+			expect(revision.void).toEqual(true);
+		});
+
+		test('should update a revision\'s status if the body of the request is boolean', async () => {
+			await agent.patch(`${route(voidRevision._id)}?key=${users.tsAdmin.apiKey}`)
+				.send({ void: false }).expect(templates.ok.status);
+
+			const res = await agent.get(`/v5/teamspaces/${teamspace}/projects/${project.id}/containers/${modelWithRev._id}/revisions?showVoid=true&key=${users.tsAdmin.apiKey}`)
+				.expect(templates.ok.status);
+			const revision = res.body.find((r) => r._id === voidRevision._id);
+			expect(revision.void).toEqual(false);
+		});
+	});
```

Comment:
missing tests:
- throw invalidArguments if it has anything extra in the payload (e.g. someone trying to change the tag of the revision)
- user with only viewer/commenter access right should not be able to update the revision